requirements/constraints.txt

```diff
@@ -40,7 +40,7 @@ drf-yasg<1.17.1
 # The team that owns this package will manually bump this package rather than having it pulled in automatically.
 # This is to allow them to better control its deployment and to do it in a process that works better
 # for them.
-edx-enterprise==3.3.26
+edx-enterprise==3.4.26
```

Comment:
If we did a minor version bump from 3.3.x to 3.4.x, should the version be bumped from 3.3.26 to 3.4.0? I think we skipped 3.4.0 to 3.4.25 ;)

Reply:
whoops yes should have done that, but now that the version is on pypi it might be more confusing to release a new version with a lower version number?

Reply:
Yeah, since it's already been published, it's probably OK but good to keep in mind for the next minor version bump 😄